### PR TITLE
Corporate partners export auth

### PR DESCRIPTION
**Pull request by darokel**
This locks down `/en/corporate/export-partners` behind http auth. Accompanying PR for the creds are placed in `scripts`. 

https://github.com/moneyadviceservice/scripts/pull/265

### CODE REVIEW

File: features/support/env.rb [hunk 1]
```diff
@@ -94,5 +95,11 @@
   create(:corporate_partner)
 end
 
+Before('@auth-required') do
+  allow(Authenticable).to receive(:username)  { ENV['AUTH_USERNAME'] }
+  allow(Authenticable).to receive(:password)  { ENV['AUTH_PASSWORD'] }
```
Comment (munckymagik) [inline features/support/env.rb]: Oh! Will these methods (username, password) on Authenticable not return the value of these env vars when when testing? What I mean is, it looks like these stubs reimplement the same thing as the real module does, is that not the case?

**Comment by urfolomeus** [comment]
:+1:

File: features/support/env.rb (hunk 1 above)
Comment (darokel) [inline features/support/env.rb]: Argh yeah I forgot to replace those with some explicit fake data I was going to pass in. Thanks for spotting this.

**Comment by munckymagik** [comment]
:+1: once the stubbing thing is sorted.

**Pull request merged**